### ISSUE TITLE
Occasionals not filters out in grid view

### ISSUE THREAD

**Issue by PaulieGregory** [Gig-o-Matic/GO3#381]

**Comment by aaronoppenheimer** [comment]
Hi Paul! Will take a look.

**Comment by aaronoppenheimer** [comment]
Yes, this will take some thinking to fix. The old gig-o has the opposite problem: occasional members are *never* shown even if they have said they'll be there. It would be easy enough to replicate the old behavior, but really the grid needs a rethink.

Should I revert the behavior to "never show occasionals ever" for now at least?

**Comment by PaulieGregory** [comment]
I think I am actually the only use the grid and I use it to see how many
people are not chiming in which I seldom do as it is a futile battle. I
don’t think any Good Troublers use it other than me. It is significant as
we have a comical number of occasionals having no mechanism for deleting
members even if: “we haven’t heard from you in 6 years and you live in
Cambodia ”.  For me the old way made more sense based on my use but…
This is a low level quirk and not a problem IMO.

**Comment by aaronoppenheimer** [comment]
I woke up in the middle of the night with a reasonably simple way to put a toggle in there that just turns occasionals on and off. Will get to it!

**Comment by PaulieGregory** [comment]
Amazing!!! Thanks
